### ISSUE TITLE
The problem that the shape and image of a specific sheet disappear when saving a file

### ISSUE THREAD

**Issue by jongba** [PHPOffice/PhpSpreadsheet#2621]
```php
<?php

require __DIR__ . '/vendor/autoload.php';

$reader = new \PhpOffice\PhpSpreadsheet\Reader\Xlsx();
$spreadsheet = $reader->load($origin_file);

$writer = new \PhpOffice\PhpSpreadsheet\Writer\Xlsx($spreadsheet);
$writer->setPreCalculateFormulas(false);
$writer->setOffice2003Compatibility(true);
$new_file = './file_uploads/report_file/' . $file_info_idx.'_tmp.'.$file_ext;
$writer->save($new_file);
```

- original file

![image](https://user-images.githubusercontent.com/100351772/155512806-adf64a0f-17f5-4b9b-b571-0e501e97d4b7.png)

- write new file

![image](https://user-images.githubusercontent.com/100351772/155512771-07d2b6b7-e55b-4857-978b-2ea54e3de259.png)

As above, when a figure or image of the original file is saved as a new file, the figure or image is lost.

Is there something I'm missing out on?

**Comment by oleibman** [comment]
Please upload the original file.

**Comment by jongba** [comment]
[1046.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/8137430/1046.xlsx)

**Comment by oleibman** [comment]
Thank you for the file. I can confirm your problem. The images that are missing all seem to be "shapes" rather than "drawings", but not all shapes are missing (e.g. on sheet 2). I don't know anything about shape files, except that they have been involved in other issues. It could take a while to figure out what is happening, and what should be happening instead.

**Comment by beomheechoi** [comment]
Only certain shapes are kept and the rest of the shapes are lost.
What code should I write to keep it?

**Comment by oleibman** [comment]
I don't think the problem is in your code. I think the problem can be solved only by better handling of shape files by PhpSpreadsheet, which I think will not be a quick or easy change.

**Comment by MarkBaker** [comment]
Correct; shape files aren't supported at all by PhpSpreadsheet; we'd have to build that support from the basics upwards

**Comment by jongba** [comment]
So what should we do?

please tell me how

**Comment by MarkBaker** [comment]
You live with it until we can implement it; or work out how to implement it yourself and submit a PR

**Comment by rikikuswo** [comment]
@MarkBaker is there any solution for this problem, i also experienced it, my shape is missing

**Comment by ikibiki** [comment]
drawing or shape is removed ☹️
